packages/apollo-links/src/retryLink.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { RetryLink } from "@apollo/client/link/retry";
+
+export const retryLink = new RetryLink({
+  delay: {
+    initial: 500,
+    max: 5000,
+    jitter: true
+  },
+  attempts: {
+    max: 2,
```

Comment:
How does the retry work? When it reaches max attempts, can we fall back to backup dictionary?

Reply:
Retry will start working when catch network errors, and will retry to send request from `auth` to `http request`, this will try to get next agreement. When the retry reach max attempts, will try to use the backup dict